testing/trino-testing-containers/src/main/java/io/trino/testing/containers/TestContainers.java

```diff
@@ -55,4 +61,21 @@ public static String getPathFromClassPathResource(String resourcePath)
                 // and mounted from there.
                 .getResolvedPath();
     }
+
+    public static void exposeFixedPorts(GenericContainer<?> container)
+    {
+        checkState(System.getenv("CONTINUOUS_INTEGRATION") == null, """
+                Exposing fixed ports should not be used in regular test code. This could break parallel test execution.
+                This method is supposed to be invoked from local development helpers only e.g. QueryRunner.main(),
+                hence it should never run on CI
+                """);
+
+        ImmutableList<PortBinding> portBindings = container.getExposedPorts().stream()
+                .map(exposedPort -> new PortBinding(bindPort(exposedPort), new ExposedPort(exposedPort)))
+                .collect(toImmutableList());
+
+        container.withCreateContainerCmdModifier(cmd -> cmd
+                .withHostConfig(new HostConfig()
+                        .withPortBindings(portBindings)));
```

Comment:
Can it be that `HostConfig` is already set?
why not use `cmd.getHostConfig()`?

Reply:
Smth like this:
`        container.withCreateContainerCmdModifier(cmd -> cmd
                .withHostConfig(cmd.getHostConfig() == null ? new HostConfig() : cmd.getHostConfig()
                        .withPortBindings(portBindings)));`
?

Reply:
can `cmd.getHostConfig()` be null?
do you know why in PTL we never have to do `new HostConfig()`?

Reply:
So maybe, like this 
`        container.withCreateContainerCmdModifier(cmd -> cmd
                .withHostConfig(requireNonNull(cmd.getHostConfig(), "hostConfig is null")
                        .withPortBindings(portBindings)));`

Reply:
`requireNonNull(cmd.getHostConfig(), "hostConfig is null")` -- that's definitely better